src/scripts/main.js

```diff
@@ -1 +1,17 @@
 'use strict';
+
+const imageList = document.querySelector('.gallery__list');
+const largeImage = document.querySelector('#largeImg');
+const links = document.querySelectorAll('.list-item__link');
+
+for (const link of links) {
+  link.addEventListener('click', (eventLink) => {
+    eventLink.preventDefault();
+  });
+}
+
+imageList.addEventListener('click', (eventImg) => {
+  const image = eventImg.target.closest('.gallery__img');
```

Comment:
```suggestion
  const image = eventImg.target.closest('.list-item__link');
```
Ищи ссылку, чтобы сразу у нее взять href